packages/manager/apps/web/client/app/hosting/cdn/order/hosting-cdn-order.service.js

```diff
@@ -0,0 +1,75 @@
+import find from 'lodash/find';
+
+export default class HostingCdnOrderService {
+  /* @ngInject */
+  constructor(OrderService) {
+    this.OrderService = OrderService;
+  }
+
+  async getCatalogAddon(
+    ovhSubsidiary,
+    serviceOption,
+  ) {
+    const { addons } = await this.OrderService
+      .getProductPublicCatalog(ovhSubsidiary, 'webHosting');
+
+    const addon = find(addons, { planCode: serviceOption.planCode });
+
+    if (!addon) {
+      throw new Error('No addon found');
```

Comment:
Alright ! I like it, but, it's kind of generic ! Could you add more data ? For example the name of the planCode